### ISSUE TITLE
Add support of Rails 8.0.0

### ISSUE THREAD

**Issue by kwent** [mainio/ratonvirus#37]
## Add support for Rails 8.0.0

```
Because every version of ratonvirus-clamby depends on ratonvirus ~> 0.4.0
  and ratonvirus >= 0.4.0 depends on activesupport ~> 7.0,
  every version of ratonvirus-clamby requires activesupport ~> 7.0.
  ```

**Comment by rylanb** [comment]
Any update here? Blocked on our Rails 8 upgrade w/ this one.

**Comment by ahukkanen** [comment]
> Any update here? Blocked on our Rails 8 upgrade w/ this one.

Update: we are aware of this, not a priority right now.

**Comment by rylanb** [comment]
> > Any update here? Blocked on our Rails 8 upgrade w/ this one.
> 
> Update: we are aware of this, not a priority right now.

Thanks for the reply!

**Comment by ahukkanen** [comment]
New version with Rails 8 support is now released:
https://github.com/mainio/ratonvirus/releases/tag/v0.6.0
https://rubygems.org/gems/ratonvirus/versions/0.6.0
